### DISCUSSION TITLE
Showcasing AR Buttons on Non-AR Desktops

### DISCUSSION THREAD

**Discussion by abhirami100** [google/model-viewer#4656] [Q&A] [answered]
If it's possible to display a custom augmented reality (AR) button on a desktop computer even if the desktop is not AR-enabled.

**Comment by danilokrulj** [comment]
What's stopping you from making a button? I always have two buttons, one on PC that opens up a QR code to open the page on mobile, and a regular AR button for mobile devices, I just toggle their display settings depending on the device that is being used.

**Reply by abhirami100** [reply]
Currently, after scanning the QR code, there's an additional step of clicking a button to access AR view. I'd like to set, similar to mobile devices, the AR view opens  right after clicking the button. How can I achieve this direct transition to AR view on PC?

**Reply by danilokrulj** [reply, marked as answer]
You mean open AR on mobile from PC in one step (like scanning QR code and open AR)? If that's the case I think you will have no luck with opening AR without clicking the button first, I wanted to implement same behavior but WebXR wont start without user interaction (user has to click button) because of security reasons.
